certified-connectors/Peltarion/apiDefinition.swagger.json

```diff
@@ -0,0 +1,111 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PeltarionConnector",
```

Comment:
Unfortunately, the word "connector" isn't allowed in the title of the connector. Could that be addressed? Perhaps simply "Peltarion" for the title?

Reply:
Of course, thanks for the suggestion. It's now updated.